README.md

```diff
@@ -46,76 +47,169 @@ servers := client.GetServerList()
 For creating and updating/patching objects in gridscale, it will be required to use the respective CreateRequest and UpdateRequest types. For creating an SSH-key that would be SshkeyCreateRequest and SshkeyUpdateRequest. Here an example:
 
 ```go
-requestBody := gsclient.IpCreateRequest {
+requestBody := gsclient.IPCreateRequest {
 	Family: 6,
-	Name:   "IpTest",
+	Name:   "IPTest",
 }
 
-client.CreateIp(requestBody)
+client.CreateIP(requestBody)
 ```
 
 What options are available for each create and update request can be found in the source code. After installing it should be located in: 
 ```
 ~/go/src/github.com/gridscale/gsclient-go
 ```
+## Examples
+Examples on how to use each resource can be found in the examples folder:
+* Firewall (firewall.go)
+* IP (ip.go)
+* ISO-image (isoimage.go)
+* Loadbalancer (loadbalancer.go)
+* Network (network.go)
+* Object Storage (objectstorage.go)
+* PaaS service (paas.go)
+* Server (server.go)
+* Storage (storage.go)
+* Storage snapshot (snapshot.go)
+* Storage snapshot schedule (snapshotschedule.go)
+* SSH-key (sshkey.go)
+* Template (template.go)
 
 ## Implemented API Endpoints
 
 Not all endpoints have been implemented in this client, but new ones will be added in the future. Here is the current list of implemented endpoints and their respective function written like endpoint (function):
```

Comment:
is it still valid? we did implement them!

Reply:
Marketplace is still missing.